src/renderers/webvr/WebVRManager.js

```diff
@@ -59,8 +59,8 @@ function WebVRManager( renderer ) {
 		if ( isPresenting() ) {
 
 			var eyeParameters = device.getEyeParameters( 'left' );
-			var renderWidth = eyeParameters.renderWidth;
-			var renderHeight = eyeParameters.renderHeight;
+			var renderWidth = eyeParameters.renderWidth * scope.renderScaling;
```

Comment:
should `scope.renderScaling` not be used when a VR device is not presenting but the WebVR renderer is `enabled`?

Reply:
@cvan - hmm...I'm not sure that it should be.  In the case that we're not presenting we use the set size of the renderer, which the user can set.  The problem when we're presenting is the eye parameters may be smaller then the actual resolution, so we need to try to scale it if we want 1:1 pixel mapping.  I'm happy to update it for the non presenting case though if you think it would be good to be consistent here.

Reply:
@cvan - following up on this, what do you think? Should I modify the else clause as well or just use the canvas element size as set?

Reply:
@cvan what action do you want taken here? I'm happy to make the change you suggest but want to make sure you're ok with the side effect, i.e. it looks like in the non presenting case we just respect the canvas width/height that was specified, should we restore this with a vr scaling factor applied to it? That feels a bit strange.